crates/virtio-device/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "virtio-device"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
I think this is updating the wrong Cargo.toml.

Reply:
Right!